manifests/base/jupyter-habana-notebook-imagestream.yaml

```diff
@@ -31,6 +31,7 @@ spec:
         opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.8"},{"name":"Habana","version":"1.10"}]'
         opendatahub.io/notebook-python-dependencies: '[{"name":"Boto3","version":"1.26"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp-tekton","version":"1.5"},{"name":"Matplotlib","version":"3.6"},{"name":"Numpy","version":"1.23"},{"name":"Pandas","version":"1.5"},{"name":"Scikit-learn","version":"1.2"},{"name":"Scipy","version":"1.10"},{"name":"Tensorflow","version":"2.12.0"},{"name":"PyTorch","version":"2.0.1"},{"name":"Elyra","version":"3.15"}]'
         openshift.io/imported-from: quay.io/modh/odh-habana-notebooks
+        opendatahub.io/workbench-image-recommended: 'true'
```

Comment:
I thought Habana and TrustyAI notebooks are not supposed to have recommended

Reply:
great observation.

Reply:
I agree with habana, but  for the TrustyAI we need recommendation annotation